app/build.gradle

```diff
@@ -2,14 +2,15 @@ plugins {
     id "com.android.application"
     id "kotlin-android"
     id "kotlin-parcelize"
+    id "com.mikepenz.aboutlibraries.plugin"
 }
 
 android {
     compileSdk 33
 
     defaultConfig {
         applicationId "com.github.muellerma.mute_reminder"
-        minSdk 14
+        minSdk 21
```

Comment:
Which library requires this?

Reply:
this one `implementation "com.mikepenz:aboutlibraries:$about_libraries_version"`